File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int k =0;
        int n = nums.size();
        if(n==1){
            return 1;
        }
        for(int i =0 ; i<n-1; i++){
            if(nums[i]!=nums[i+1]){
                nums[k++]=nums[i];
            }
           
        }
        nums[k++] = nums[n-1];
        return k;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let newArray = [];
    let n = nums.length;

    for(let i =0 ; i<n ; i++){
        if(!newArray.includes(nums[i]))
        newArray.push(nums[i]);
    }

    nums.length = 0;
    nums.push(...newArray);

    return newArray.length;
};